admin/client/src/components/FieldHolder/FieldHolder.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import SilverStripeComponent from 'lib/SilverStripeComponent';
+
+function fieldHolderWrapper(Field) {
```

Comment:
Please remember to create a README.md next time when creating a new component, even if its just a single sentence describing the component. Fixing that myself now